README.md

```diff
@@ -20,53 +20,35 @@ Contributing to FoundationDB can be in contributions to the code base, sharing y
 
 ### Binary downloads
 
-Developers interested in using the FoundationDB store for an application can get started easily by downloading and installing a binary package. Please see the [downloads page](https://www.foundationdb.org/download/) for a list of available packages.
+Developers interested in using FoundationDB can get started by downloading and installing a binary package. Please see the [downloads page](https://www.foundationdb.org/download/) for a list of available packages.
 
 
 ### Compiling from source
 
 Developers on an OS for which there is no binary package, or who would like
 to start hacking on the code, can get started by compiling from source.
 
-Currently there are two build systems: a collection of Makefiles and a
-CMake-based build system. Both of them should currently work for most users,
-and CMake should be the preferred choice as it will eventually become the only
-build system available.
+The official docker image for building is `foundationdb/foundationdb-build`. It has all dependencies installed. Use `-DBOOST_ROOT=/opt/boost_1_72_0/` in your cmake command. To build outside the official docker image you'll need at least these dependencies:
```

Comment:
It might be better to break out the docker directions into their own section:
```
docker run -it foundationdb/foundationdb-build
git clone https://github.com/apple/foundationdb.git
(cd foundation-db; git checkout release-x.y ) # optional
mkdir fdb-build
cd fdb-build
cmake -G Ninja -DBOOST_ROOT=opt/boost_1_72_0/ ../foundationdb/
ninja
```

Reply:
Uggh.  With the above directions, I'm getting these errors on my machine:
```
CMAKE_C_COMPILER_RANLIB:FILEPATH=/opt/rh/devtoolset-8/root/usr/bin/gcc-ranlib
[root@15546edf041f build]# ninja
[382/767] Building CXX object fdbclient/CMakeFiles/fdbclient.dir/NativeAPI.actor.g.cpp.o
FAILED: fdbclient/CMakeFiles/fdbclient.dir/NativeAPI.actor.g.cpp.o 
/opt/rh/devtoolset-8/root/usr/bin/g++  -DBOOST_ERROR_CODE_HEADER_ONLY -DBOOST_SYSTEM_NO_DEPRECATED -DNO_INTELLISENSE -DUSE_UCONTEXT -I/foundationdb -I. -isystem /usr/local/include -isystem /opt/boost_1_72_0 -O3 -DNDEBUG   -DCMAKE_BUILD -ggdb -fno-omit-frame-pointer -Wno-pragmas -Wno-attributes -Wno-error=format -Wunused-variable -Wno-deprecated -fvisibility=hidden -Wreturn-type -fPIC -DHAVE_OPENSSL -std=gnu++17 -MD -MT fdbclient/CMakeFiles/fdbclient.dir/NativeAPI.actor.g.cpp.o -MF fdbclient/CMakeFiles/fdbclient.dir/NativeAPI.actor.g.cpp.o.d -o fdbclient/CMakeFiles/fdbclient.dir/NativeAPI.actor.g.cpp.o -c fdbclient/NativeAPI.actor.g.cpp
g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.

[384/767] Building CXX object fdbserver/CMakeFiles/fdbserver.dir/LogRouter.actor.g.cpp.o
FAILED: fdbserver/CMakeFiles/fdbserver.dir/LogRouter.actor.g.cpp.o 
/opt/rh/devtoolset-8/root/usr/bin/g++  -DBOOST_ERROR_CODE_HEADER_ONLY -DBOOST_SYSTEM_NO_DEPRECATED -DNO_INTELLISENSE -DUSE_UCONTEXT -I/foundationdb -I. -I/foundationdb/bindings/c -Ibindings/c -Ifdbserver/workloads -I/foundationdb/fdbserver/workloads -isystem /usr/local/include -isystem /opt/boost_1_72_0 -O3 -DNDEBUG   -DCMAKE_BUILD -ggdb -fno-omit-frame-pointer -Wno-pragmas -Wno-attributes -Wno-error=format -Wunused-variable -Wno-deprecated -fvisibility=hidden -Wreturn-type -fPIC -DHAVE_OPENSSL -std=gnu++17 -MD -MT fdbserver/CMakeFiles/fdbserver.dir/LogRouter.actor.g.cpp.o -MF fdbserver/CMakeFiles/fdbserver.dir/LogRouter.actor.g.cpp.o.d -o fdbserver/CMakeFiles/fdbserver.dir/LogRouter.actor.g.cpp.o -c fdbserver/LogRouter.actor.g.cpp
g++: fatal error: Killed signal terminated program cc1plus
compilation terminated.
```
It's probably an out of memory error (I have 16GB + 11 cores dedicated to docker).  Should we add a blurb about ninja using all the CPUs (and therefore, sometimes too much DRAM)?

Reply:
I didn't feel confident writing ready-to-copy-and-paste docker instructions, but I think it makes sense having them. A blurb about memory usage also sounds good, I added that.